src/panels/lovelace/cards/hui-map-card.ts

```diff
@@ -209,6 +209,10 @@ class HuiMapCard extends LitElement implements LovelaceCard {
       this._attachObserver();
     }
 
+    if (!this._config.aspect_ratio) {
+      return;
+    }
```

Comment:
It should default to 1:1 when not defined. `aspect_ratio` is optional.

Reply:
So the type should be updated.

Reply:
The type was updated to be optional